WordPress/build.gradle

```diff
@@ -144,7 +148,10 @@ android {
             buildConfigField "String", "PUSH_NOTIFICATIONS_APP_KEY", '"com.jetpack.android"'
 
             manifestPlaceholders = [magicLinkScheme:"jetpack"]
+
         }
+        addBuildConfigFieldsFromPrefixedProperties(jetpack, properties, 'wp') // Inherit same properties used from WP
+        addBuildConfigFieldsFromPrefixedProperties(jetpack, properties, 'jp') // Add or overwrite with Jetpack-specific properties
```

Comment:
There's a risk here that having the first call declare `buildConfigField "String" "SENTRY_DSN" "the wp dsn"` then the second call declare `buildConfigField "String" "SENTRY_DSN" "the jp dsn"` that we'd get a warning telling us about the redefinition 🤔 

I tried to search if this could be a problem, and found [this SO question] which mentions that this could lead to a warning. But when I ran `./gradlew` with those changes I didn't see any warning, so maybe we're good? 🤔

Reply:
Alternative idea: have the method take an array of prefixes, and make its implementation iterate over each to build a dictionary/hash of field=>value first – thus letting the last value override previous ones in that Hash – and only then iterate again on the Hash to set the `buildConfigField`, ensuring there would be no duplicate that way. Not sure it's worth the trouble though?